contents/docs/error-tracking/cutting-costs.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Cutting error tracking costs
+sidebar: Docs
+showTitle: true
+---
+
+We aim to be significantly cheaper than our competitors. In addition to our [ridiculously cheap pricing](/pricing), below are tips to reduce your error tracking costs:
+
+## Configure exception autocapture
+
+Exception autocapture is a powerful feature that captures every exception your users encounter. It can also capture more than you need. To reduce which exceptions that are captured, you can configure which types of exceptions are autocaptured in the JS SDK config. By default we capture all unhandled errors and rejections.
```

Comment:
```suggestion
By default, we capture all unhandled errors and rejections. This can capture more than you need. To reduce which exceptions that are captured, you can configure which types of exceptions are autocaptured in the JS SDK config like this: 
```